psydac/api/tests/build_domain.py

```diff
@@ -211,6 +206,6 @@ def build_pretzel(domain_name='pretzel', r_min=None, r_max=None):
         ]
 
 
-    domain = set_interfaces(domain, interfaces)
+    domain = create_domain(domains, interfaces, domain_name)
```

Comment:
As above.

Reply:
done in [d66a68e](https://github.com/pyccel/psydac/pull/309/commits/d66a68ed7d030e41dc86823c805ed685b78d906e)